placeSpecificContent/theSpot/moneyTree/entityScripts/moneyTreeBankerClient.js

```diff
@@ -0,0 +1,197 @@
+// moneyTreeBankerClient.js
+
+//  Created by Mark Brosche on 11-8-2018
+//  Copyright 2018 High Fidelity, Inc.
+//
+//  Distributed under the Apache License, Version 2.0.
+//  See the accompanying file LICENSE or http://www.apache.org/licenses/LICENSE-2.0.html
+//
+/* global EventBridge Users AccountServices Agent Avatar */
+
+(function (){
+    // get userData
+    // spawn Overlays for clients with ID == giverID
+    // delete after overlay after click event or after timer expires.
+    // send data to google sheet
+    var SECRETS = Script.require(Script.resolvePath('../moneyTreeURLs.json'));
```

Comment:
this is dangerous because it is running on a client script accessible to anyone. hopefully not a huge deal.

Reply:
I made a second JSON for the channels, so the AC script will be the only one that accesses the urls and lists and can be hidden better.